.circleci/generate_config.py

```diff
@@ -221,8 +221,20 @@ def filter_tests(args, tests, enterprise):
         tests = filter(one_filter, tests)
     return list(tests)
 
+def get_size(size, arch):
+    aarch64_sizes = {
+        "small": "arm.medium",
+        "medium": "arm.medium",
+        "medium+": "arm.large",
+        "large": "arm.large",
+        "xlarge": "arm.xlarge"
+    }
+    if arch == "aarch64":
```

Comment:
why param this in? shouldn't 
```
IS_ARM = platform.processor() == "arm" or platform.processor() == "aarch64"
```
be enough here?

Reply:
No, this script runs just once on some system, but generates the jobs for all architectures.